inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/include/matchers/matchers_manager.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "pugixml.hpp"
+#include "ngraph/node.hpp"
+#include "single_op.hpp"
+
+namespace SubgraphsDumper {
+class Matcher;
+
+class MatchersManager {
+public:
+    typedef std::map<std::string, std::function<Matcher::Ptr()>> registry_map;
+    typedef std::map<std::string, Matcher::Ptr> matchers_map;
+
+    explicit MatchersManager(const std::string &cfg_path = "");
```

Comment:
`""` -> `{}`. By compiler optimization will be the same but `{}` is more general rule.

Reply:
Fixed